compiler/noirc_evaluator/src/ssa/opt/normalize_value_ids.rs

```diff
@@ -179,19 +179,6 @@ impl IdMaps {
             Value::NumericConstant { constant, typ } => {
                 new_function.dfg.make_constant(*constant, typ.clone())
             }
-            Value::Array { array, typ } => {
```

Comment:
Is there no replacement for `MakeArray` in this pass? E.g. with `Value::Instruction`?

Reply:
This pass doesn't need to handle them explicitly at all now since Instruction::MakeArray will already be handled along with other instructions by Instruction::map